README.md

```diff
@@ -48,6 +48,7 @@
 * [Vality](https://github.com/jeengbe/vality)
 * [yup](https://github.com/jquense/yup)
 * [zod](https://github.com/vriad/zod)
+* [valibot](https://github.com/fabian-hiller/valibot)
```

Comment:
Oops, sorry, just noticed this. The list is in alphabetical order. Could you please sort it? Thanks!!

Reply:
Sure! Fixed now 👍